syntax = "proto2";
package dhc;


message arena_list_t {
    optional uint64 guid                    = 1;
    repeated uint64 player_guids            = 2;
    repeated string player_names            = 3;
    repeated int32 player_levels            = 4;
    repeated int32 player_icons             = 5;
    repeated int32 player_segments          = 6;
    repeated int32 is_npc                   = 7;
    repeated int32 player_powers            = 8;
    optional uint64 last_time               = 9;
}
